stacks/services/APIGateway.ts

```diff
@@ -0,0 +1,93 @@
+import {
+    AuthorizationType,
+    AwsIntegration,
+    MethodOptions,
+    PassthroughBehavior,
+    Resource,
+} from "aws-cdk-lib/aws-apigateway";
+import { PolicyStatement, Role, ServicePrincipal } from "aws-cdk-lib/aws-iam";
+import { ApiGatewayV1Api, Stack, Bucket } from "sst/constructs";
+
+export const createSiriApi = (stack: Stack, siriSXBucket: Bucket): void => {
+    const apiGateway = new ApiGatewayV1Api(stack, "cdd-siri-sx-api", {
+        cdk: {
+            restApi: {
+                restApiName: `cdd-siri-sx-api-${stack.stage}`,
+                description: "API to retrieve Siri SX XML data",
+            },
+        },
+    });
+
+    const bucketResource: Resource = apiGateway.cdk.restApi.root.addResource("{folder}");
```

Comment:
Since this endpoint will only ever be retrieving the single xml document, it might be better if we just have a static `/siri-sx` endpoint rather than having `/{bucket}/{object}` like it is now. Then the `/siri-sx` endpoint would always return the contents of the `SIRI-SX.xml` file

Reply:
done

Reply:
done